modules/security-deploy-signature.adoc

```diff
@@ -38,9 +38,9 @@ docker:
 
 In this example, the Red Hat Registry, `access.redhat.com`, is the signature
 server that provides signatures for the `docker` transport type. Its URI is
-defined in the `sigstore` parameter. You might name this file 
+defined in the `sigstore` parameter. You might name this file
 `/etc/containers/registries.d/redhat.com.yaml` and use the Machine Config
 Operator to
 automatically place the file on each node in your cluster. No service
 restart is required since policy and `registries.d` files are dynamically
-loaded by the container runtime. 
+loaded by the container runtime.
```

Comment:
Looks like this file is just picking up removing extra whitespace at the end of these lines. What do you think about undoing the hard line wraps in this module?

Reply:
Yeah, we'll need to do some follow-up work there, but that's out of scope for this PR. Thanks!